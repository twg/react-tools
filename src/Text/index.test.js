/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'
import { H1, H2, H3, H4, H5, H6, P } from './'

describe('Text Components', () => {
  it('matches H1 snapshot', () => {
    let component = renderer.create(
      <div>
        <H1>Testing value </H1>
      </div>
    )

    expect(component.toJSON()).toMatchSnapshot()
  })

  it('matches H2 snapshot', () => {
    let component = renderer.create(
      <div>
        <H2>Testing value </H2>
      </div>
    )

    expect(component.toJSON()).toMatchSnapshot()
  })

  it('matches H3 snapshot', () => {
    let component = renderer.create(
      <div>
        <H3>Testing value </H3>
      </div>
    )

    expect(component.toJSON()).toMatchSnapshot()
  })

  it('matches H4 snapshot', () => {
    let component = renderer.create(
      <div>
        <H4>Testing value </H4>
      </div>
    )

    expect(component.toJSON()).toMatchSnapshot()
  })

  it('matches H4 snapshot', () => {
    let component = renderer.create(
      <div>
        <H4>Testing value </H4>
      </div>
    )

    expect(component.toJSON()).toMatchSnapshot()
  })

  it('matches H5 snapshot', () => {
    let component = renderer.create(
      <div>
        <H5>Testing value </H5>
      </div>
    )

    expect(component.toJSON()).toMatchSnapshot()
  })

  it('matches H6 snapshot', () => {
    let component = renderer.create(
      <div>
        <H6>Testing value </H6>
      </div>
    )

    expect(component.toJSON()).toMatchSnapshot()
  })

  it('matches P snapshot', () => {
    let component = renderer.create(
      <div>
        <P>Testing value </P>
      </div>
    )

    expect(component.toJSON()).toMatchSnapshot()
  })
})
