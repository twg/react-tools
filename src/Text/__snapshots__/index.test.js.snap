// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Components matches H1 snapshot 1`] = `
<div>
  <h1
    className="h1"
  >
    Testing value 
     
  </h1>
</div>
`;

exports[`Text Components matches H2 snapshot 1`] = `
<div>
  <h2
    className="h2"
  >
    Testing value 
     
  </h2>
</div>
`;

exports[`Text Components matches H3 snapshot 1`] = `
<div>
  <h3
    className="h3"
  >
    Testing value 
     
  </h3>
</div>
`;

exports[`Text Components matches H4 snapshot 1`] = `
<div>
  <h4
    className="h4"
  >
    Testing value 
     
  </h4>
</div>
`;

exports[`Text Components matches H4 snapshot 2`] = `
<div>
  <h4
    className="h4"
  >
    Testing value 
     
  </h4>
</div>
`;

exports[`Text Components matches H5 snapshot 1`] = `
<div>
  <h5
    className="h5"
  >
    Testing value 
     
  </h5>
</div>
`;

exports[`Text Components matches H6 snapshot 1`] = `
<div>
  <h6
    className="h6"
  >
    Testing value 
     
  </h6>
</div>
`;

exports[`Text Components matches P snapshot 1`] = `
<div>
  <p
    className="p"
  >
    Testing value 
     
  </p>
</div>
`;
